x-pack/plugins/ingest_manager/server/services/epm/packages/get.ts

```diff
@@ -49,6 +50,28 @@ export async function getPackages(
   return packageList;
 }
 
+// Get package names for packages which cannot have more than one package config on an agent config
+// Assume packages only export one config template for now
+export async function getLimitedPackages(options: {
+  savedObjectsClient: SavedObjectsClientContract;
+}): Promise<string[]> {
+  const { savedObjectsClient } = options;
+  const allPackages = await getPackages({ savedObjectsClient });
+  const installedPackages = allPackages.filter(
```

Comment:
I think we can skip the get & filter by calling [`getPackageSavedObjects`](https://github.com/elastic/kibana/blob/2685654cdc67f6630a6c96feee09967c1f78034f/x-pack/plugins/ingest_manager/server/services/epm/packages/get.ts#L52). I don't know if it'll be a drop-in replacement, but it will start with the installed packages instead of getting all from the registry & filtering.

Not a blocker, just a comment

Reply:
It will end up being the same number of calls because `getPackageSavedObjects` just returns the saved objects, which does not contain the necessary PackageInfo fields (from the registry) to detect if the package is limited or not